server/paymentProviders/transferwise/index.ts

```diff
@@ -1,8 +1,14 @@
 import uuidv4 from 'uuid/v4';
 
 import * as transferwise from '../../lib/transferwise';
+import cache from '../../lib/cache';
+import models from '../../models';
 import { Quote } from '../../types/transferwise';
 
+export const blackListedCurrencies = [
+  'BRL', // Businesses customers are not supported yet.
```

Comment:
Not sure I understand this one

Reply:
This is a TransferWise limitation, it only allows bank transfers between private parts.
That's why I'm currently blacklisting that.

Reply:
The right way to do is to check if the host has a private or a business account and allow if it's private. But I don't think we should worry about that right now.